scripts/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -105,6 +106,41 @@ public static BuildTargetResult LocateStage0(BuildTargetContext c)
         }
 
         [Target]
+        public static BuildTargetResult ExpectedBuildArtifacts(BuildTargetContext c)
```

Comment:
not a fan of `ExpectedBuildArtifacts` as a name. Not too sure what the action taken will be.

Most of the other targets seem to be using a verb in the name as well.